using-timescaledb/continuous-aggregates.md

```diff
@@ -160,123 +275,91 @@ something like:
 ```sql
 SELECT * FROM conditions_summary_daily
 WHERE max - min > 1800
-  AND bucket >= '2018-01-01' AND bucket < '2018-04-01'
+  AND bucket >= '2020-01-01' AND bucket < '2020-04-01'
 ORDER BY bucket DESC, device_id DESC LIMIT 20;
 ```
 
+---
+
+### Real-Time Aggregates [](real-time-aggregates)
+
+Real-time aggregates are a capability (first introduced in TimescaleDB 1.7)
+whereby querying the *continuous aggregate view* will then compute
+fully up-to-date aggregate results by combining the materialized
+partial aggregate with recent data from the hypertable that has yet to
```

Comment:
we have removed "how this works" section. Without that the reader does not know what is the materialized partial aggregate. 
Adding a couple of sentences on what is stored in the materialization  should suffice as an introduction to the partial aggregate.

Reply:
+1

Reply:
I think this whole section is bloated, complicated, and confusing. I suggest folding this into the intro. Probably best left for a separate PR though.

Reply:
Leaving it for now. I will handle it as a separate PR.